package.json

```diff
@@ -7,7 +7,7 @@
     "dev": "webpack-dev-server",
     "start": " serve -s dist",
     "heroku-postbuild": "rm -rf dist && NODE_ENV=production webpack -p --progress --colors",
-    "test": "jest --config=testConfig/jest.config.json",
+    "test": "jest --config=testConfig/jest.config.json --coverage",
```

Comment:
Should we worry about `yarn run test:watch` generating a report on each file save?